modules/storage-api/src/main/java/org/apache/ignite/internal/storage/engine/MvTableStorage.java

```diff
@@ -162,4 +163,48 @@ default IndexStorage getOrCreateIndex(int partitionId, UUID indexId) {
      * @throws StorageException If an error has occurred during the destruction of the storage.
      */
     void destroy() throws StorageException;
+
+    /**
+     * Prepares the partition storage for rebalancing: makes a backup of the current partition storage and creates a new one.
+     *
+     * <p>This method must be called before each full rebalance of the partition storage, so that in case of errors or cancellation of the
```

Comment:
```suggestion
     * <p>This method must be called before every full rebalance of the partition storage, so that in case of errors or cancellation of the
```

Reply:
Fix it